lld/ELF/Thunks.cpp

```diff
@@ -402,6 +402,21 @@ class AVRThunk : public Thunk {
   void addSymbols(ThunkSection &isec) override;
 };
 
+// Hexagon CPUs need thunks for <<FIXME TBD>>
```

Comment:
Please fix FIXME

Reply:
Ok, fixed.  Is it useful/expected to describe the range of the relocations without the thunks here like `AVRThunk` does?  If so, I'll add that here.